src/k8s/pkg/utils/database.go

```diff
@@ -28,6 +30,99 @@ func GetClusterConfig(ctx context.Context, state *state.State) (types.ClusterCon
 	return clusterConfig, nil
 }
 
+// GetUserFacingClusterConfig returns the public cluster config.
+func GetUserFacingClusterConfig(ctx context.Context, state *state.State) (apiv1.UserFacingClusterConfig, error) {
```

Comment:
An important note for this. This is not `utils`: `utils` are supposed to be stand-alone helpers.

I see why this is here, same as the others in the same file. But we have some them in `api/impl` as well. The problem is that this does not allow importing the `pkg/utils` package from `pkg/k8sd/database` code, which is not ideal.

It's OK to deal with this at a later time.